public/app/plugins/panel/news/NewsPanel.tsx

```diff
@@ -1,185 +1,52 @@
-import { css, cx } from '@emotion/css';
-import React, { PureComponent } from 'react';
-import { Unsubscribable } from 'rxjs';
+import React, { useEffect } from 'react';
```

Comment:
Decided to convert this to functional so I can use a hook to make the api call. Happy to revert it to class component if needed